app/script/search/SearchRepository.js

```diff
@@ -56,6 +56,49 @@ z.search.SearchRepository = class SearchRepository {
     this.logger = new z.util.Logger('z.search.SearchRepository', z.config.LOGGER.OPTIONS);
   }
 
+  /**
+   * Search for a user in the given user list and given a search term
+   * Doesn't sort the results and keep the initial order of the given user list
+   * @param {string} term - the search term
```

Comment:
Let's clean up the JSDoc a bit.

- end name with .
- add empty like below name
- prefix info